integrations/ivy/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "ivy",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve --hmr",
+    "test": "ng build --prod",
+    "pretest": "yarn --pure-lockfile",
+    "postinstall": "cpx -v -C \"../../@ngxs/**/*\" \"node_modules/@ngxs/\""
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "9.0.0-next.3",
+    "@angular/common": "9.0.0-next.3",
+    "@angular/compiler": "9.0.0-next.3",
+    "@angular/core": "9.0.0-next.3",
+    "@angular/forms": "9.0.0-next.3",
+    "@angular/platform-browser": "9.0.0-next.3",
+    "@angular/platform-browser-dynamic": "9.0.0-next.3",
+    "@angular/router": "9.0.0-next.3",
+    "rxjs": "file:../../node_modules/rxjs",
+    "tslib": "file:../../node_modules/tslib",
```

Comment:
@arturovt I know that you asked @splincode to make these relative paths... why do we want this as opposed to the explicit versions (specific to this version of angular we are testing with ivy)?

Reply:
... is this so that we are copying the same dependencies as used by `@ngxs/store` and plugins?

Reply:
I can use absolute if you needed

Reply:
Just wanting to know the reasoning behind @arturovt 's suggestion here

Reply:
Answering your question :) Ivy doesn't require specific versions of those packages as they can be used with View Engine either. Eventually there is no reason to specify versions explicitly as those packages can be provided via local paths.

Reply:
Ok, it makes sense to me! Thanks.